pkg/kubelet/kubelet.go

```diff
@@ -1011,6 +1011,15 @@ type Kubelet struct {
 	//    as it takes time to gather all necessary node information.
 	nodeStatusUpdateFrequency time.Duration
 
+	// syncNodeStatusMux is a lock on updating the node status, because this path is not thread-safe.
+	syncNodeStatusMux sync.Mutex
+
+	// updatePodCIDRMux is a lock on updating pod CIDR, because this path is not thread-safe.
+	updatePodCIDRMux sync.Mutex
+
+	// updateRuntimeUpMux is a lock on updating runtime, because this path is not thread-safe.
```

Comment:
Either change the name of the variable to `updateRuntimeUpMux` or fix the godoc.

Reply:
done